### DISCUSSION TITLE
chia перестал определять плоты

### DISCUSSION THREAD

**Discussion by Dmitriy363** [Chia-Network/chia-blockchain#5781] [Support]
Система Windows 10 x64 bit стояла на HDD в разделе С , раздел С заполнился временными файлами при создании плотов я взял SSD на 1Tb и скопировал систему с раздела С на SSD с помощью программы Paragon Hard Disk Manager Advanced 17.13.1 RePack by elchupacabra , запустил программу с SSD , прошла синхронизация но плоты не определяет , подскажите что сделать что-б определила Chia плоты .

**Comment by firmicus** [comment]
Type : chia plots add -d [NEW DIRECTORY]

**Comment by BrandtH22** [comment]
@Dmitriy363 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
